src/app/transactions/create/page.jsx

```diff
@@ -1,8 +1,135 @@
+"use client";
 import "@/app/globals.css";
+import { React } from "react";
+import { useRouter } from "next/navigation";
+import { useContractCreateStore } from "@/stores/contractCreateStore";
+import InfoText from "@/components/common/InfoText";
+import InputField from "@/components/common/InputField";
+import InputAccountField from "@/components/common/InputAccountField";
+import MainButton from "@/components/common/MainButton";
 
 /* 새 계약서 생성하기 */
 const CreatePage = () => {
-  return <div></div>;
-};
+  const router = useRouter();
+  const { contract, setField } = useContractCreateStore();
+
+  /* TODO: 더미 데이터 수정 필요 */
+  const bankOptions = [
+    { value: "우리은행", label: "우리은행" },
+    { value: "신한은행", label: "신한은행" },
+    { value: "국민은행", label: "국민은행" },
+  ];
+
+  /* 로딩 및 데이터 상태 관리 */
+  // const [isLoading, setIsLoading] = useState(false);
+
+  const handleNextStep = () => {
+    const requiredFields = [
+      "title",
+      "start_date",
+      "end_date",
+      "amount",
+      "client_name",
+      "client_phone",
+      "account_number",
+      "bank",
+    ];
+
+    const isFormValid = requiredFields.every(
+      (key) => contract[key] && contract[key].trim() !== "",
+    );
+
+    if (!isFormValid) {
+      alert("모든 필수 정보를 입력해주세요.");
+      return;
+    }
 
+    const startDate = new Date(contract.start_date);
+    const endDate = new Date(contract.end_date);
+
+    if (isNaN(startDate) || isNaN(endDate)) {
+      alert("날짜 형식이 올바르지 않습니다.");
+      return;
+    }
+
+    if (endDate < startDate) {
+      alert("종료일은 시작일 이후여야 합니다.");
+      return;
+    }
+
+    router.push("/transactions/create/preview");
+  };
+
+  return (
+    <div className="px-8 py-4">
+      <InfoText
+        mainTexts={["진행될 계약 내용을", "작성해주세요"]}
+        subText="워켓이 계약서를 만들어 거래 페이지를 제공합니다!"
+        subTextColor="gray"
+      ></InfoText>
+      <div className="flex flex-col justify-center items-center">
+        <div>
+          <InputField
+            question="계약명"
+            placeholder="계약명을 작성해주세요"
+            value={contract.title || ""}
+            onChange={(e) => setField("title", e.target.value)}
+          ></InputField>
```

Comment:
태그 안에 들어갈 내용이 없으면 단일 태그 형태로 해주어도 됩니당
```suggestion
          <InputField
            question="계약명"
            placeholder="계약명을 작성해주세요"
            value={contract.title || ""}
            onChange={(e) => setField("title", e.target.value)}
          />
```

Reply:
우와 맞네요~ 단일태그로 수정하니 한결 깔끔해요.. 쵝오👍🏻